charts/app-config/values-staging.yaml

```
@@ -697,6 +697,48 @@ govukApplications:
         - name: WEB_CONCURRENCY
           value: '4'
 
+  - name: content-store-postgresql-branch
+    repoName: content-store-postgresql-branch
+    helmValues:
+      <<: *content-store
+      rails:
+        createKeyBaseSecret: false
+        # use the same secret as the mongo content-store, it will make the
+        # eventual switchover easier and reduce toil
+        secretKeyBaseName: content-store-rails-secret-key-base
+      sentry:
+        createSecret: false  # Sentry DSNs are per repo.
+        dsnSecretName: content-store-sentry
+      extraEnv:
+        - name: SENTRY_ENVIRONMENT
+          value: "postgresql-integration"
```

s/integration/staging/ ? (similarly for the draft one below)
```suggestion
          value: "postgresql-staging"
```

---

oooh, good catch, thanks :+1: